packages/web5/lib/src/dids/did_web/did_web.dart

```diff
@@ -97,7 +97,7 @@ class DidWeb {
     }
 
     final String documentUrl = Uri.decodeFull(did.id.replaceAll(':', '/'));
-    Uri? didUri = Uri.tryParse('https://$documentUrl');
+    Uri? didUri = Uri.tryParse('http://$documentUrl');
```

Comment:
@ethan-tbd i think this was refactored out at some point. not sure when but, here's what we can add back to support both: https://github.com/TBD54566975/web5-dart/blob/01f420ed8000c2538f9d21b3ed99de8f8c2253dc/packages/web5/lib/src/dids/did_web/did_web.dart#L20-L26